src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -180,8 +180,9 @@ def _check_health_environment(ignore_errors, yes):
 # Check DNS lookup and access to challenge endpoint
 def _get_registry_status(login_server, registry_name, ignore_errors):
     import socket
+    from requests.exceptions import SSLError
```

Comment:
NIT: can you import this after you import requests? :D

Reply:
Yes, that would be better.